compiler/rustc_resolve/src/lib.rs

```diff
@@ -881,6 +881,16 @@ impl<'a> NameBindingData<'a> {
             invoc_parent_expansion.is_descendant_of(self_parent_expansion);
         !(certainly_before_other_or_simultaneously || certainly_before_invoc_or_simultaneously)
     }
+
+    fn determined(&self) -> bool {
```

Comment:
This function needs a comment describing its purpose and its relations with `NameResolution::binding`.

Maybe also a FIXME saying that the logic in `NameResolution::binding` and here should be merged/unified.